src/index.js

```diff
@@ -1,29 +1,32 @@
 import './style.css';
 
+import * as Task from './task.js';
+
+Task.displayTasks();
 const data = [{
-        index: 1,
-        description: 'Attend a meeting',
-        completed: true,
-    }, {
-        index: 4,
-        description: 'Go shopping in the supermarket',
-        completed: true,
-    }, {
-        index: 2,
-        description: 'Go to the gym',
-        completed: false,
-    }, {
-        index: 3,
-        description: 'Go to the club and have some drinks',
-        completed: true,
-    },
+  index: 1,
+  description: 'Attend a meeting',
+  completed: true,
+}, {
+  index: 4,
+  description: 'Go shopping in the supermarket',
+  completed: true,
+}, {
+  index: 2,
+  description: 'Go to the gym',
+  completed: false,
+}, {
+  index: 3,
+  description: 'Go to the club and have some drinks',
+  completed: true,
+},
 
 ];
 let toDoItem = '';
 const itemList = document.querySelector('#list-container');
 
 data.forEach((item, index) => {
-    toDoItem += `
+  toDoItem += `
     <li class="items" id="${index}">
     <input type="checkbox" id="check-b" name="checkbox" value="">
     <p id="item-list">${item.description}</p> <a href="#"><i id="fa-ellipsis-v" class="fas fa-ellipsis-v"></i></a>
```

Comment:
- [ ] I believe that you don't need this block of code in this file because you have basically handled all these features in your `task` and `tasks` files and you are already calling `Task.displayTasks()` here at the top. I would recommend to remove this to keep your code "DRY" and clean.